inlang/source-code/versioned-interfaces/message/src/interface.ts

```diff
@@ -58,6 +58,7 @@ export const Variant = Type.Object({
 export type Message = Static<typeof Message>
 export const Message = Type.Object({
 	id: Type.String(),
+	alias: Type.Record(Type.String(), Type.String()),
```

Comment:
Shouldn't the alias by undefineable to ensure backwards compatibility similar to `settings.featureFlags`?

Reply:
Objects from type Message currently only exist in memory - even if one plugin uses the AST to store them - this one will be replaced by the import export before the feature flag is active which will take care of that.
 
Making the alias type non optional saves us an extra check if alias is present and the overall structure is cleaner - i would say - we keep it like that.